### PR TITLE
market app update

### PR DESCRIPTION
**Pull request by mmattel**
This PR fixes: https://github.com/owncloud/docs/issues/449 (Document new install and upgrade options)
Plus some additional minor fixes.

### CODE REVIEW

File: modules/administration_manual/pages/configuration/server/occ_app_command.adoc [hunk 1]
```diff
@@ -189,7 +189,7 @@ Parameters for short form options will be written either directly after the opti
 sudo -u www-data ./occ files:scan -p "user_x/files/folder"  
 ....
 
-[[brute-force-protection]]
+[[ocapps_brute-force-protection]]
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/occ_app_command.adoc]: Why don't you just remove these, and use [automatic anchors](https://asciidoctor.org/docs/user-manual/#automatic-anchors) instead?

File: modules/administration_manual/pages/configuration/server/occ_app_command.adoc [hunk 2]
```diff
@@ -755,7 +773,58 @@ sudo -u www-data occ market:install twofactor_backup_codes
 sudo -u www-data occ market:install -l /mnt/data/richdocuments-2.0.0.tar.gz
 ....
 
-[[password-policy]]
+[[uninstall-an-application]]
+=== Uninstall an Application
+
+To uninstall an application use following commands:
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/occ_app_command.adoc]: s/use/use the/g

File: modules/administration_manual/pages/configuration/server/occ_app_command.adoc [hunk 3]
```diff
@@ -755,7 +773,58 @@ sudo -u www-data occ market:install twofactor_backup_codes
...
+
+[width="80%",cols="30%,70%",]
+|===
+| `ids` |  Ids of the apps
+|===
+
+[[list-apps-from-the-marketplace]]
+=== List Apps From The Marketplace
+
+This command lists apps available on the marketplace. It returns the ids if the apps.
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/occ_app_command.adoc]: Can you put the second sentence onto the next line, pls?

File: modules/administration_manual/pages/configuration/server/occ_app_command.adoc [hunk 4]
```diff
@@ -755,7 +773,58 @@ sudo -u www-data occ market:install twofactor_backup_codes
...
+
+....
+sudo -u www-data occ market:list
+....
+
+[[upgrade-an-application]]
+=== Upgrade an Application
+
+Install new app versions if available on the marketplace by using following commands
```
Comment (settermjd) [inline modules/administration_manual/pages/configuration/server/occ_app_command.adoc]: Pls end this sentence with a colon.

**Comment by mmattel** [comment]
Fixes done

**Comment by mmattel** [comment]
Merge that one, I am just fixing that as part of another PR which I am working on.

File: modules/administration_manual/pages/configuration/server/occ_app_command.adoc (hunk 1 above)
Comment (settermjd) [reply modules/administration_manual/pages/configuration/server/occ_app_command.adoc]: What's the reason for not using automatic anchors?

**Comment by mmattel** [comment]
After docs went life, I saw that there were some small things to improve, added with the latest squashed commit.

**Pull request merged**